manifests/kustomize/third-party/argo/upstream/manifests/quick-start/base/artifact-repositories-configmap.yaml

```diff
@@ -11,6 +19,16 @@ data:
       secretKeySecret:
         name: my-minio-cred
         key: secretkey
-kind: ConfigMap
-metadata:
-  name: artifact-repositories
+  my-key: |
+    archiveLogs: true
+    s3:
+      bucket: my-bucket
+      endpoint: minio:9000
+      insecure: true
+      accessKeySecret:
+        name: my-minio-cred
+        key: accesskey
+      secretKeySecret:
+        name: my-minio-cred
+        key: secretkey
+  empty: ""
```

Comment:
new line?

Reply:
Note these files are pulled from argo without touch, so I'd recommend avoid making any changes.

Reply:
Yes that makes sense.